meson.build

```diff
@@ -41,6 +41,14 @@ has_backends = false
 # Third party files.
 includes += include_directories('third_party', is_system: true)
 
+# Android support
+if get_option('android')
+  add_global_link_arguments('-llog', language:'cpp')
+  if get_option('opencl')
```

Comment:
It may be better to move this part after opencl detection. Say somewhere in the `if get_option('opencl') and has_opencl` block.
Also better combine it with a check that the .so file exists. Since the code will only run on android an `if run_command('test', '-f', '/system/lib64/liblzma.so').returncode() == 0` will probably be sufficient.

Reply:
Agree. But the -llog line should stay there as it's needed for BLAS back-end too.

Reply:
Yes, my comment was intended from line 47 down.